stylus/components/wizard.styl

```diff
@@ -0,0 +1,168 @@
+@require '../tools/mixins'
+@require '../settings/breakpoints'
+@require '../settings/palette'
+@require '../components/forms'
+
+// Wizard layout
+$wizard
+    position fixed
+    top 0
+    left 0
+    height 100%
+    display flex
+    justify-content center
+    align-items center
+    width 100%
+    background-color white
+    color charcoalGrey
+    text-align center
+    transition height ease 300ms
```

Comment:
This is surprising to have a transition on height since it will trigger a reflow/repaint each time. I think using transition only on properties that are managed by the composer is a better practice.

Reply:
This is needed for iOS (when we'll resize the wrapper on keyboard open). 
What do you call "properties that are managed by the composer" exactly?

Reply:
I meant css properties whose changes do not affect layout or repainting and can be completely carried oud by the compositor thread on the GPU.

> Animations made with JavaScript are sometimes called imperative animations, and those made with CSS are called declarative animations. CSS animations are handled by the browser’s compositor thread rather than the main thread responsible for painting and styling. Consequently, such animations are unaffected by the main thread’s more expensive tasks. *Of course, animations that trigger the paint or layout events will require work from the main thread, which negates the benefits of using CSS animations.*

Emphasis mine.

The height property will cause layout changes, and these mean animations that are not very fluid.

https://www.keycdn.com/blog/animation-performance
https://csstriggers.com/transform

Reply:
Ok but still, I want to animate the height so I don't see what else I can do.

Reply:
Sometimes it is not necessary to animate the height, the opacity and/or scaleX/scaleY can be used. Maybe it is not necessary here.